### PR TITLE
Start multiple docker containers

### PR DESCRIPTION
**Pull request by matthewdunbar**

### CODE REVIEW

File: src/main/scala/com/lifeway/sbt/testdocker/StartTestDocker.scala [hunk 1]
```diff
@@ -27,10 +28,14 @@ object StartTestDocker {
...
         Process(arg).run()
+        var attempts = 0
+        val r        = attempts = attempts + 1
+
         do {
           streamz.log.info(s"Waiting for testdocker to boot on port $hostPort")
           Thread.sleep(250)
-        } while (!isHostPortActive(hostPort))
+          attempts = attempts + 1
+        } while (!isHostPortActive(hostPort) && attempts <= 240)
```
Comment (rmmeans) [inline src/main/scala/com/lifeway/sbt/testdocker/StartTestDocker.scala]: what does 240 represent?

Comment (matthewdunbar) [reply src/main/scala/com/lifeway/sbt/testdocker/StartTestDocker.scala]: I experimented with something a long time ago and forgot I'd made that change. I thought it was on master. I'll remove it.

**Comment by rmmeans** [comment]
I'm good with the change. can you update the README for usage and bump the version to 1.1.0? I'm not sure how the version was previously set...

**Review by jordanfjellman — DISMISSED** [review]

**Comment by matthewdunbar** [comment]
I updated the README. Versioning is handle by tags so I'll make sure to tag it appropriately.

**Review by jbellars — APPROVED** [review]

**Review by jbellars — CHANGES_REQUESTED** [review]
... did this version number need to be incremented?
README
For sbt 1.0:
```
addSbtPlugin("com.lifeway" % "sbt-testdocker" % "1.0.0")
```

**Review by jordanfjellman — APPROVED** [review]

**Comment by rmmeans** [comment]
@jbellars was right. The readme for what version needs to be updated too. Can someone hit this? @matthewdunbar @jordanfjellman @davidhulse

**Pull request merged**